doc/source/reference/potentialdehnenbar.rst

```diff
@@ -1,5 +1,5 @@
 Dehnen bar potential
-========================
+====================
```

Comment:
For future reference, it would be better to not make any changes to files in a PR that you don't actually need to change. I suspect your editor has changed the length of these title headings (the ===) automatically. It doesn't matter much here, but in general this would be better to avoid (you can restore files that you didn't want to change if you notice them with ``git status`` by doing ``git checkout -- FILE``).